package.json

```diff
@@ -10,9 +10,15 @@
     "lint: fix": "./node_modules/.bin/eslint --ext .js,.jsx src/ --fix",
     "build:css": "postcss src/assets/css/styles.css -o src/assets/css/main.css",
     "watch:css": "postcss src/assets/css/styles.css -o src/assets/css/main.css",
-    "test": "jest --collect-coverage",
+    "test": "cross-env jest --collect-coverage --env=jsdom",
     "test:watch": "jest --watch"
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "npm run lint && npm test",
+      "pre-push": "npm run lint && npm test"
```

Comment:
you don't need both pre push and pre commit. Add only one

Reply:
Add only pre commit hook. Check staged commits. You need to run lint and tests only on the changed files.